### ISSUE TITLE
Documentation to create and test visualization of new blog post

### ISSUE THREAD

**Issue by camipacifici** [spacetelescope/mast-blog#3]
**Is your feature request related to a problem? Please describe.**
When a blog post is written, how do we check how it looks before merging it?

**Describe the solution you'd like**
Probably using a fork it is possible to compile the blog without posting it publicly? Need to try.

**Describe alternatives you've considered**
No other alternatives yet.

**Additional context**
None

**Comment by ttdu** [comment]
One thought: if users can't set up the actions on their fork, [obsidian](https://obsidian.md/) is great for previewing.

<img width="719" alt="screenshot of obsidian" src="https://github.com/user-attachments/assets/fd6c9ce5-5a72-44aa-9f94-f491a50c8a47" />

**Comment by ttdu** [comment]
SOLVED: this is possible in a forked environment. It is necessary to change the github-pages branch to the feature branch containing the article, but after that it is straightforward to see new changes

<img width="1109" alt="Image" src="https://github.com/user-attachments/assets/facd844b-61b3-4225-a4ba-e1760590bdb7" />

**Comment by ttdu** [comment]
Of course, it would be nice to type this up somewhere in "contributing"...
